.github/workflows/build.yaml

```diff
@@ -75,6 +91,7 @@ jobs:
             CTEST_OUTPUT_ON_FAILURE=1 make test
```

Comment:
Don't we use `ctest` for this?

Reply:
I think this should be fine, since `cmake` commands generate a `Makefile test` command that uses ctest

Reply:
But that doesn't work in windows right?

Reply:
This is a Linux build